.github/workflows/pr.yml

```
@@ -49,6 +16,7 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
```

```suggestion
          node-version: ${{ matrix.node-version }}
          cache: 'yarn'
```

If you want to use the new cache feature that comes with `setup-node@v2` and higher, use this suggestion.